src/docs/framework-integration/angular.md

```diff
@@ -43,18 +47,28 @@ top-most-directory/
                 └── public-api.ts
 ```
 
-#### Creating an Angular Component Library
+This guide uses Lerna for the monorepo, but you can use other solutions such as Nx, TurboRepo, etc.
 
-> If you already have an Angular component library, skip this section.
+To use Lerna with this walk through, globally install Lerna:
 
-The first time you want to create the component wrappers, you will need to have an Angular library package to write to.
+```bash
+npm install --global lerna
+# or if you are using yarn
+yarn global add lerna
+```
 
-Using Angular's CLI, generate a workspace and a library for your Angular component wrappers:
+#### Creating a Monorepo
+
+> If you already have a monorepo, skip this section.
 
 ```bash
-npx -p @angular/cli ng new angular-workspace --no-create-application
-cd angular-workspace
-npx -p @angular/cli ng generate library component-library
+# From your top-most-directory/, initialize a workspace
+lerna init
+
+# install typescript and node types
+npm install typescript @types/node --save-dev
+# or if you are using yarn
+yarn add typescript @types/node --dev
```

Comment:
I think when doing something like this it's probably a good idea to use a monorepo so you can do things like run an end-to-end build from one spot, manage publishing, etc.

However when following the guide at present we don't actually do much with the monorepo after initializing it, and it doesn't instruct the reader to put the two different projects (the stencil project and the angular project) into the `packages` subdirectory where lerna expects packages to be.

So I'm wondering, does it make sense to re-factor things so that the Stencil component library and the angular thingy are nested under `packages`, commands are run from the top-level project directory using `lerna`, etc? Or should we just scrap it, is there a standard, angular-ey way of doing this? not sure!

Reply:
I noted an issue with linking below